pages.pt_BR/linux/btrfs-restore.md

```diff
@@ -0,0 +1,24 @@
+# btrfs restore
+
+> Tenta salvar arquivos de um sistema de arquivos btrfs danificado.
+> Mais informações: <https://btrfs.readthedocs.io/en/latest/btrfs-restore.html>.
+
+- Restaura todos os arquivos de um sistema de arquivos btrfs para um determinado diretório:
```

Comment:
Idealmente convém usar o infinitivo para as descrições:

```suggestion
- Restaurar todos os arquivos de um sistema de arquivos btrfs para um determinado diretório:
```

Infelizmente não estamos ainda a usar esse padrão de forma consistente, mas assim vamos caminhando nessa direção :)

Reply:
Então, eu já tinha contribuído uma vez pro tldr, e eles pediram pra deixar de forma imperativa, e não no infinitivo. Então não sei como proceder aqui...

Reply:
Pois é, lamento a confusão. O problema vem dos verbos em inglês serem conjugados de forma diferente de em português. Enquanto em português é habitual usar o infinitivo para listar instruções de forma concisa e sem referência ao sujeito (fazer X; adicionar Y; abrir Z; etc.), no inglês essa função é assumida pelo imperativo (do X; add Y; open Z; etc.).

Se fosse usado o infinitivo, não funcionaria em inglês ("To do X", etc.); igualmente, usar o imperativo em português não se adequa bem a este tipo de instrução, porque ou se teria que assumir que o sujeito omisso é o leitor (e nesse caso a conjugação seria na segunda pessoa, o que se torna um pouco "agressivo", p.ex. "tu, faz isto"), ou então o sujeito é o programa, e aí o tempo verbal certo seria o presente do indicativo, porque não estamos de facto a ordenar ao programa que faça isto ou aquilo. É uma coincidência infeliz que o imperativo da segunda pessoa (neste caso, "Restaura [tu] ...") seja idêntico ao presente do indicativo na terceira pessoa (neste caso, "[O programa] restaura ..."). 

Enfim, isto tudo para dizer que a tradução direta das orientações editoriais do tldr-pages, que originalmente foram escritas para o inglês, não funciona necessariamente para as outras línguas, daí a necessidade de orientações específicas para cada língua. Mas isto não foi ainda estabelecido claramente no style guide, infelizmente. E como a maior parte dos maintainers do projeto não falam português, houve essa tentativa de tradução direta nas contribuições passadas, com boas intenções mas cujo resultado é esta mistura inconveniente. Isso é algo que tenho no meu "backlog" há imenso tempo... :sweat_smile:

Reply:
Entendi... Realmente, tradução é algo bastante complicado. Porém acredito que você esteja certo mesmo, pelo domínio da língua que aparenta ter (e que eu certamente não tenho, mesmo sendo nativo hahaha). O mantenedor que sugeriu as modificações para o imperativo não era nativo da língua portuguesa, mas sim do espanhol, ele é um dos mantenedores principais do tldr e acredito que seja argentino.

Reply:
Ainda sim, quanto ao uso do imperativo eu penso que cada começo de frase descreve o que o comando faz. Nesse sentido, o imperativo é o mais indicado, pois posso falar "Esse comando ... faz, restaura, etc...". Não seria muito bem o imperativo, pois o imperativo do verbo fazer é "faça" (como uma ordem mesmo), mas pro nosso contexto o adequado seria "faz", pois explica o que o comando, bem, faz.

Reply:
Exacto: se quiséssemos, podíamos usar o (presente do) indicativo, para descrever o que o comando faz. Mas como disse acima, em contexto instrutivo, faz mais sentido usar o infinitivo, que é mais comum e portanto evoca a leitura (e ação consequente) apropriada.